extensions/blocks/story/player/components/bullet.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { createElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+
+export default function Bullet( { index, progress, onClick } ) {
+	return (
+		<button
+			key={ index }
+			className="wp-story-pagination-bullet"
+			aria-label={ `Go to slide ${ index }` }
```

Comment:
We'll need to translate this as well.

Reply:
That's one advantage of going `@wordpress/element` instead of preact, I can load other gutenberg packages now such as `@wordpress/i18n` :). The downside is that the app is tied to the gutenberg/jetpack ecosystem now